common-lib/src/main/scala/com/gu/workflow/lib/LogConfig.scala

```diff
@@ -14,20 +14,13 @@ import net.logstash.logback.encoder.LogstashEncoder
 import net.logstash.logback.layout.LogstashLayout
 import org.slf4j.{LoggerFactory, Logger => SLFLogger}
 import play.api.libs.json.Json
-import play.api.{Configuration, Logger => PlayLogger}
 
 import scala.util.Try
 
-case class LogStashConf(host: String, port: Int, enabled: Boolean)
-
 object LogConfig extends AwsInstanceTags {
```

Comment:
should this be a class that takes in a `config` to be more similar to other things in this project?